students/woogyeong/westagram/users/views.py

```diff
@@ -33,4 +33,24 @@ def post(self, request):
             return JsonResponse({'message': "CREATED"}, status=201)
         except KeyError:
             return JsonResponse({'message': 'KEY_ERROR'}, status=400)
-        
\ No newline at end of file
+        
+class LogInView(View):
+    def post(self, request):
+        data     = json.loads(request.body)
+        email    = data['email']
+        password = data['password']
+
+        try:
+            user    = User.objects.get(email=email)
+            userID  = user.user_id
+            
+            if not User.objects.filter(email=email, password=password).exists():
+                return JsonResponse({'message': 'INVALID_USER'}, status=401)
+            
+            if User.objects.filter(email=email, password=password).count() == 1:
+                return JsonResponse({'message': f"SUCCESS! user_id : '{userID}' successfully logged in"}, status=200)
```

Comment:
만약 어떤 user가 로그인 했는지를 보내고 싶으면 dictionary 구조로 만들어서 보내주시면 됩니다.
```suggestion
                result = {
                    'id'      : user.id,
                    'user_id' : user.user_id,
                    'name'    : user.name
                }
                return JsonResponse({'result':result }, status=200)
```

Reply:
경훈님 감사합니다! 이런 식으로 리턴을 나오게 하고 싶었어요